readme.md

```diff
@@ -332,7 +336,54 @@ algolia exportrules -a <algoliaAppId> -k <algoliaApiKey> -n <algoliaIndexName> -
 
 - `<outputPath>`path must include file name.
 
-### 10. Transfer Index | `transferindex`
+### 10. Add Synonyms | `addsynonyms`
+
+##### Description:
+
+Import a local CSV or JSON file of synonyms to an Algolia index. Some public synonym files can be downloaded and used from the [synonyms directory](#synonyms) of this repository.
+
+##### Usage:
+
+```shell
+algolia addsynonyms -a <algoliaAppId> -k <algoliaApiKey> -n <algoliaIndexName> -s <sourceFilepath> -p <batchSynonymsParams>
+```
+
+##### Options:
+
+- `<algoliaAppId>` | Required
+- `<algoliaApiKey>` | Required
+- `<algoliaIndexName>` | Required
+- `<sourceFilepath>` | Required | Path to a CSV or JSON file containing an array of synonyms objects.
+- `<batchRulesParams>` | Optional | JSON object containing options passed to `batchSynonyms()` [method](https://www.algolia.com/doc/api-reference/api-methods/batch-synonyms/).
```

Comment:
I think this is a typo and we wanted `batchSynonymsParams`

Reply:
indeed, thanks!